src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebResponseObject.CoreClr.cs

```diff
@@ -41,6 +41,11 @@ public partial class WebResponseObject
             }
         }
 
+        /// <summary>
+        /// gets the RelationLink property
+        /// </summary>
+        public Dictionary<string, string> RelationLink { get; set; }
```

Comment:
Created https://github.com/PowerShell/PowerShell/issues/3842 for the `Headers` issue